c++/src/capnp/BUILD.bazel

```diff
@@ -74,6 +74,10 @@ cc_library(
         "ez-rpc.h",
         "persistent.capnp.h",
         "reconnect.h",
+        "message.h",
```

Comment:
ez-rpc.h is using message.h and other headers... but they were not included.

Reply:
Er, also, `ez-rpc.h` was removed a long time ago?